test/acceptance/cli-test.js

```diff
@@ -853,6 +885,10 @@ describe('ember-template-lint executable', function () {
   function run(args, options = {}) {
     options.reject = false;
     options.cwd = options.cwd || project.path('.');
-    return execa.sync(require.resolve('../../bin/ember-template-lint.js'), args, options);
+
+    // for windows we need to call node <script.js>
+    // however the node method call hangs sometimes on linux, but works smoothly on windows
+    let runFunc = process.platform === 'win32' ? execa.node : execa.sync;
+    return runFunc(require.resolve('../../bin/ember-template-lint.js'), args, options);
```

Comment:
Instead of using `execa.node`, can give a try to the following method: `execa('node', [require.resolve('../../bin/ember-template-lint.js'), ...args], options)`?

Indeed, it seems that the `shell` option cannot be used with `execa.node` (see https://github.com/sindresorhus/execa#execanodescriptpath-arguments-options). I wonder if it's not the reason why the method call hangs on linux.